ios/Classes/PspdfkitPlugin.m

```diff
@@ -178,5 +219,39 @@ - (void)unlockWithPasswordIfNeeded:(PSPDFDocument *)document dictionary:(NSDicti
     }
 }
 
+- (UIBarButtonItem *)barButtonItemFromString:(NSString *)barButtonItem forViewController:(PSPDFViewController *)pdfController {
+    if ([barButtonItem isEqualToString:@"closeButtonItem"]) {
+        return pdfController.closeButtonItem;
+    } else if ([barButtonItem isEqualToString:@"outlineButtonItem"]) {
+        return pdfController.outlineButtonItem;
```

Comment:
These name all match buttons 1:1, let's use KVC for that + check type so we are sure it's a bar button?

Reply:
I used the same implementation code from our React Native wrapper.

In there, I used KVO, but we had a discussion (see https://github.com/PSPDFKit/react-native/pull/233#discussion_r292384158) and we decided not to do it.

Reply:
@steipete I'd like to be consistent between our wrappers, so I'd leave this as-is. WDYT?

Reply:
I'm merging this as-is and will open a follow up PR if needed.